sshuttle/ssnet.py

```diff
@@ -625,8 +551,9 @@ def too_full(self):
     def uwrite(self, buf):
         if self.mux.too_full:
             return 0  # too much already enqueued
-        if len(buf) > 65535:
-            buf = buf[:65535]
+        room_left = self.mux.room_left()
+        if len(buf) >= room_left:
+            buf = buf[:room_left]
```

Comment:
Does this mechanism work well with multiplexing long-lived high-bandwidth connections with short-lived connections? E.g. download a large video file on one workspace and try to browse the web with another workspace.

Reply:
Yes, it does.  @lucbertin tested it.

Reply:
There is no issues that I can see with this.  I tested the scenarios that you have mentioned and performance was good.  We are sending either the same amount or more data at the same frequency.  On a very busy system performance is better.  Less waste.